.github/workflows/tests.yml

```diff
@@ -121,7 +155,7 @@ jobs:
         run: "composer install --no-interaction --no-progress"
 
       - name: "Tests"
-        run: "make tests-levels"
+        run: "php vendor/bin/phpunit --filter '${{ matrix.test-filter }}' --group levels"
```

Comment:
I just did some local testing and in this format PHPUnit will still execute all data providers from all tests. It would be good to also add the file in which the filtered test is. I think we can get even faster that way.

See the comparison:

```
time vendor/bin/phpunit --filter 'PHPStan\\Type\\UnionTypeTest::testSorting'
15.7s

time vendor/bin/phpunit tests/PHPStan/Type/UnionTypeTest.php --filter 'PHPStan\\Type\\UnionTypeTest::testSorting'
1.1s
```

Reply:
indeed, it got even faster now :)